_scripts/update-external-docs.rb

```diff
@@ -73,7 +80,7 @@ def fetch_and_add(pages, type=:source)
     file = "#{type}/#{page}"
     file += ".md" unless page.match(/\.md$/)
 
-    doc = open(url_raw).read
+    doc = URI.open(url_raw).read
```

Comment:
I snuck in a little fix here too. Calling `uri-open` in Ruby with just "open" is deprecated and throws a warning. Hence, adding `URI.` to be explicit on which open we want to use.